lib/public/views/Runs/format/formatRunDetectors.js

```diff
@@ -29,23 +29,51 @@ export const formatRunDetectors = (run, detailsModel) => {
         return '-';
     }
 
+    const _detectorTypes = detailsModel.detectorTypes.match({
```

Comment:
The convention of `_` prefix is for classes private members. Here you are not in a class, so you should simply name the variable `detectorTypes`.
However, I would suggest to name it `allDetectors` because it's actually a list of detectors, there is no such thing as detector type (please adapt the model accordingly).